src/libraries/token/BasicToken.sol

```diff
@@ -33,4 +34,12 @@ contract BasicToken is ERC20Basic {
     function balanceOf(address _owner) public constant returns (uint256 balance) {
         return balances[_owner];
     }
+
+    function getBalance(address _address) public constant returns (uint256) {
```

Comment:
This is a duplicate of `balanceOf`, so is unnecessary.  Even though `balanceOf` doesn't comply with our naming convention, I don't think it is necessary to duplicate the method _just_ to conform and it can lead to user confusion when there are multiple methods that do the same thing.